server/repository/src/db_diesel/storage_connection.rs

```diff
@@ -70,81 +112,15 @@ impl From<TransactionError<RepositoryError>> for RepositoryError {
 impl StorageConnection {
     pub fn new(connection: DBConnection) -> StorageConnection {
         StorageConnection {
-            connection,
-            transaction_level: Cell::new(0),
-        }
-    }
-    /// Executes operations in transaction. A new transaction is only started if not already in a
-    /// transaction.
-    pub async fn transaction<'a, T, E, F, Fut>(&'a self, f: F) -> Result<T, TransactionError<E>>
```

Comment:
The async version was not needed, and I think wasn't working when used in async code anyway (connection is not Sync...) so I removed it...